includes/blocks/podcast/markup.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Podcast markup
+ *
+ * @package tenup_podcasting
+ *
+ * @var array    $attributes Block attributes.
+ * @var string   $content    Block content.
+ * @var WP_Block $block      Block instance.
+ * @var array    $context    Block context.
+ */
+
+$attributes = wp_parse_args(
+	$attributes ?? [],
+	[
+		'id'                   => null,
+		'caption'              => '',
+		'displayDuration'      => false,
+		'displayShowTitle'     => false,
+		'displayEpisodeTitle'  => false,
+		'displayArt'           => false,
+		'displayExplicitBadge' => false,
+		'displaySeasonNumber'  => false,
+		'displayEpisodeNumber' => false,
+		'displayEpisodeType'   => false,
+	]
+);
+
+if ( ! $attributes['id'] ) {
+	return;
+}
+
+$post_id        = get_the_id();
+$podcast_shows  = get_the_terms( get_the_ID(), 'podcasting_podcasts' );
+$show_name      = $podcast_shows ? $podcast_shows[0]->name : '';
+$src            = get_post_meta( $post_id, 'src', true );
+$duration       = get_post_meta( $post_id, 'podcast_duration', true );
+$explicit       = get_post_meta( $post_id, 'podcast_explicit', true );
+$episode_type   = get_post_meta( $post_id, 'podcast_episode_type', true );
+$episode_number = get_post_meta( $post_id, 'podcast_episode_number', true );
+$season_number  = get_post_meta( $post_id, 'podcast_season_number', true );
+
+?>
+<div class="wp-block-podcasting-podcast-outer">
+	<div class="wp-block-podcasting-podcast__container">
+		<?php if ( has_post_thumbnail() && $attributes['displayArt'] ) : ?>
```

Comment:
If the post thumbnail is not set, then this should default to the image set on the podcast taxonomy.